<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1352</width>
    <height>858</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>800</width>
      <height>640</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>500</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>460</y>
      <width>500</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>430</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Biển số:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>470</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>710</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Mở</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_3">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>570</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_4">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>570</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>button</zorder>
   <zorder>graphicsView_2</zorder>
   <zorder>tableView</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>graphicsView_3</zorder>
   <zorder>graphicsView_4</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1352</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
